website/docs/r/cloud_aws_integrations.html.markdown

```diff
@@ -87,6 +162,13 @@ Some integration types support an additional set of arguments:
   * `tag_value` - (Optional) Specify a Tag value associated with the resources that you want to monitor. Filter values are case-sensitive.
 * `x_ray`
   * `aws_regions` - (Optional) Specify each AWS region that includes the resources that you want to monitor.
+* `api_gateway`
+  * `aws_regions` - (Optional) Specify each AWS region that includes the resources that you want to monitor.
+  * `metrics_polling_interval` - (Optional) The data polling interval in seconds.
+  * `tag_key` - (Optional) Specify a Tag key associated with the resources that you want to monitor. Filter values are case-sensitive.
+  * `tag_value` - (Optional) Specify a Tag value associated with the resources that you want to monitor. Filter values are case-sensitive.
+  * `stage_prefixes` - (Optional) Determine if extra inventory data be collected or not. May affect total data collection time and contribute to the Cloud provider API rate limit.
```

Comment:
What about the rest of the fields mentioned like app_sync, fsx etc? Is there a particular reason for excluding them?

Reply:
That's correct, since most of the other services ( AppSync, AutoScaling, Aws Fsx etc) shares the common fields, I just thought to add ApiGateway reference as it had some other additional fields.